jest.setup.js

```diff
@@ -8,6 +8,9 @@
   global.TextDecoder = TextDecoder;
 }
 
+// Fix for 'Reference Error: __static is not defined
+global.__static = '';
```

Comment:
Use `__static` for this value

Reply:
I wasn't sure if you meant to change the left or right side of the assignment, but I just tried both 
`__static = '';` and `global.__static=__static;` and both generate the reference error that `global.__static=''` fixes